cost/aws/superseded_instance/aws_superseded_instance_remediation.pt

```diff
@@ -0,0 +1,217 @@
+name "AWS Superseded Instance Remediation"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy Template is used to automatically resize AWS Instances based on user-defined standards. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/aws_superseded_instance) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Cost"
+severity "low"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_instances" do
+  label "Permissions for instances"
+  actions "rs_cm.index", "rs_cm.show"
+  resources "rs_cm.instances"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses"
+  description "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_old_instance_type" do
+  label "Disallowed Instance Type"
+  type "string"
+  description "Old Instance Type"
+end
+
+parameter "param_new_instance_type" do
+  label "Superseded Instance Type"
+  type "string"
+  description "New Instance Type"
+end
+
+parameter "param_exclude_tags" do
+  label "Exclude Tags"
+  description "list of excluded servers"
```

Comment:
This param isn't the list of excluded server.  It's a list of tags to filter out instances from being checked.